webpack.config.js

```diff
@@ -17,8 +17,8 @@ const path = require('path');
 module.exports = {
   entry: './src/index.ts',
   output: {
-    library: 'TextToSpeech',
-    filename: './text-to-speech.js',
+    library: 'texttospeech',
```

Comment:
Could this have potential to break folks?

Reply:
we are doing semver major!   but technically yes, you are right. @xiaozhenliu-gg5 let's do PascalCase here - I'll send a PR to the generator to change that.